### PR TITLE
feat(VWindow,VCarousel): **prev** and **next** slots

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
## Description
Adds slots for prev/next arrow

## Motivation and Context
fixes partially #3235

## How Has This Been Tested?
playground

## Markup:

<details>

```vue
<template>
  <v-container>
    <v-window
      reverse
      show-arrows
    >
      <template #prev="{ on, attrs }">
        <v-btn
          color="success"
          v-bind="attrs"
          v-on="on"
        >Prev</v-btn>
      </template>
      <template #next="{ on, attrs }">
        <v-btn
          color="info"
          v-bind="attrs"
          v-on="on"
        >Next</v-btn>
      </template>
      <v-window-item
        v-for="n in 3"
        :key="`card-${n}`"
        class="pa-12"
      >Lorem ipsum</v-window-item>
    </v-window>

    <v-carousel>
      <template #prev="{ on, attrs }">
        <v-btn
          color="success"
          v-bind="attrs"
          v-on="on"
        >Prev</v-btn>
      </template>
      <template #next="{ on, attrs }">
        <v-btn
          color="info"
          v-bind="attrs"
          v-on="on"
        >Next</v-btn>
      </template>
      <v-carousel-item
        v-for="i in 6"
        :key="i"
      >
        <div class="black--text">
          Slide {{ i + 1 }}
        </div>
      </v-carousel-item>
    </v-carousel>
  </v-container>
</template>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Review by nekosaur — CHANGES_REQUESTED** [review]
There's no need to repeat the slot definition or translation. Add `source: 'v-window'` to the definition in `maps/v-window.js` and then re-use them in `maps/v-carousel.js` (same way as v-data-table map re-uses v-data-iterator slots). Then translation is only necessary for `v-window.json`

File: packages/api-generator/src/locale/en/v-window.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "change": "Emitted when the component value is changed by user interaction"
   },
   "slots": {
-    "next": "Slots displaying the arrow switching to the next slide",
-    "prev": "Slots displaying the arrow switching to the previous slide"
+    "next": "Slots displaying the arrow switching to the next item",
```
Comment (nekosaur) [inline packages/api-generator/src/locale/en/v-window.json:18]:
```suggestion
    "next": "Slot displaying the arrow switching to the next item",
```

File: packages/api-generator/src/locale/en/v-window.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
+    "prev": "Slots displaying the arrow switching to the previous item"
```
Comment (nekosaur) [inline packages/api-generator/src/locale/en/v-window.json:19]:
```suggestion
    "prev": "Slot displaying the arrow switching to the previous item"
```

**Review by nekosaur — CHANGES_REQUESTED** [review]

**Review by nekosaur — APPROVED** [review]

**Review by KaelWD — APPROVED** [review]

**Comment by KaelWD** [comment]
This doesn't entirely resolve #3235 though, you still can't modify the delimiters.

**Comment by jacekkarczmarczyk** [comment]
Adding `delimiter` slots needs some more decisions and would be done in a separate PR, i've updated description

> fixes partially #3235

**Review by KaelWD — APPROVED** [review]

**Pull request merged**